csrankings.csv

```diff
@@ -8982,8 +8990,8 @@ Jonathan E. Rowe,University of Birmingham,https://www.birmingham.ac.uk/staff/pro
 Jonathan F. Buss,University of Waterloo,https://cs.uwaterloo.ca/~jfbuss,68wU4zYAAAAJ
 Jonathan Gemmell,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facultyinfo.aspx?fid=494,RIhZrvIAAAAJ
 Jonathan I. Maletic,Kent State University,http://www.cs.kent.edu/~jmaletic,n9_W4kYAAAAJ
-Jonathan K. Lazar,Towson University,http://orion.towson.edu/~jlazar,CDpfbBsAAAAJ
-Jonathan Katz,George Mason University,https://www.cs.umd.edu/~jkatz,H4rKFc8AAAAJ
+Jonathan K. Lazar,University of Maryland - College Park,https://ischool.umd.edu/faculty-staff/jonathan-lazar,CDpfbBsAAAAJ
+Jonathan Katz,University of Maryland - College Park,https://www.cs.umd.edu/~jkatz,H4rKFc8AAAAJ

```

Comment:
Web page clearly states no longer at Maryland.

Reply:
You are correct. I missed the recent affiliation of Prof. Jonathan Katz. I have corrected it to George Mason University in the csrankings.csv file.